core/src/integrationTest/java/amino/run/kernel/IntegrationTestBase.java

```diff
@@ -84,8 +85,19 @@ public static Process startOms(String ip, int port) throws java.io.IOException {
                     Integer.toString(port)
                 };
 
-        ProcessBuilder builder = new ProcessBuilder(args);
-        Process process = builder.inheritIO().start();
+        System.out.printf(
+                "Starting OMS at %s:%d with command line \'%s\'\n",
+                ip, port, StringUtils.join(args, ","));
+
+        Process process = new ProcessBuilder(args).start();
+
+        // Must read the command STDOUT otherwise command is hanging.
+        // 1.7 Java version has processBuilder.inhertIO() will read the logs and print on console.
+        StreamReader error = new StreamReader(process.getErrorStream(), "ERROR");
+        StreamReader output = new StreamReader(process.getInputStream(), "");
```

Comment:
IMO, Instead of maintaining two stream here , we can use ```redirectErrorStream``` method which will redirect Error stream to output stream. We need to just use OUTPUT stream only for messages.